server/index.js

```diff
@@ -29,6 +32,13 @@ app.use("/user", userRouter);
 app.use("/habit", verifyToken, habitRouter);
 app.use("/answer", verifyToken, answerRouter);
 
+// Error route
+app.get("/error", (req, res) => {
+  throw new Error("Intentional error");
+});
+
+app.use(unexpectedErrorHandler);
+
```

Comment:
it's nice to write this part and check the handler, but let's leave this route out of the PR.

Reply:
Removed check route